### PR TITLE
[BetterStandardPrinter] Fixes #4691 Donot convert string literal comment

### LINKED ISSUE #4691: string literal in phpdocs gets converted into a type
# Bug Report

processing code which contains a literal string in a phpdoc breaks semantics.

## Minimal PHP Code Causing Issue

https://getrector.org/demo/f2d8f721-f345-4d34-a2c9-5190c8ab388f#result

## Expected Behaviour

rector should not strip the double quotes , since this makes the literal string beeing interpreted as a type in phpdoc.

result should be the same as the input.

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

**Review by TomasVotruba — APPROVED** [review]

**Comment by samsonasik** [comment]
Rebased got the PHPStan error:

```bash
[WARNING] Internal error: Internal error: Call to a member function            
           getAttribute() on null in file                                       
           /home/runner/work/rector/rector/packages/better-php-doc-parser/src/Va
           lueObject/PhpDocNode/Doctrine/AbstractDoctrineTagValueNode.php       
           Run PHPStan with --debug option and post the stack trace to:         
           https://github.com/phpstan/phpstan/issues/new            
```

**Comment by staabm** [comment]
Really cool, thx for fixing my issue.

I had a similar issue with this comment today - maybe it can be fixed in a similar way:

https://github.com/redaxo/redaxo/blob/cf387cbeadb4ce76d45f325afb5bc388302cc5f3/redaxo/src/core/functions/function_rex_escape.php#L26

**Comment by samsonasik** [comment]
@staabm this won't works for single quote use case : 

```php
 * @psalm-param 'html'|'html_simplified'|'html_attr'|'js'|'css'|'url' $strategy
```

That may need different PR with different regex for that.

File: phpstan.neon [hunk 1]
```diff
@@ -753,6 +753,11 @@ parameters:
                 - packages/changes-reporting/src/Output/JsonOutputFormatter.php # 14
                 - src/Console/Output/RectorConfigurationFormatter.php # 13
 
+        -
+            message: '#Parameter 1 should use PhpParser\\Node\\Expr\\MethodCall type as already checked#'
+            paths:
+                - rules/nette-kdyby/src/Naming/VariableNaming.php #267
```
Comment (samsonasik) [inline phpstan.neon:759]: note:

multi call condition need to be skipped in symplify's phpstan rule:

```php
if ($var instanceof PropertyFetch) {
      $this->call($var);
}

if ($var instanceof MethodCall) {
      $this->call($var);
}
```

**Comment by samsonasik** [comment]
@TomasVotruba `symplify/phpstan-rules` dev-master seems not immediately updated when patch pushed to `symplify/simplify`

**Comment by TomasVotruba** [comment]
@samsonasik What PR do you talk about? If this one https://github.com/symplify/symplify/pull/2564, it needs to be merged first, before it gets to the `master` branch. Only the `master` is split then

**Comment by samsonasik** [comment]
This one https://github.com/symplify/symplify/pull/2563 which check next node is an identifier, that after merged to master, seems was not immediatelly splitted to phpstan-rules package

**Comment by TomasVotruba** [comment]
I see it here: https://github.com/symplify/phpstan-rules/commit/579fe06970fccd6a2647bfe4663faf5dcb00954f#diff-9c7b4bdb02497d19d66be18326ea89f8fb1b50a377e45625fc6aefe71179341d

Where is missing?

It might take 1 minute to split, then couple minutes to packagist. In some cases packagist bugged and hours to update.

**Comment by samsonasik** [comment]
Oh yeah, that seems updated now for next node check.

**Comment by samsonasik** [comment]
Still can't get the only one PHPStan error: 

```bash
[WARNING] Internal error: Internal error: Call to a member function            
           getAttribute() on null in file                                       
           /home/runner/work/rector/rector/packages/better-php-doc-parser/src/Va
           lueObject/PhpDocNode/Doctrine/AbstractDoctrineTagValueNode.php       
           Run PHPStan with --debug option and post the stack trace to:         
           https://github.com/phpstan/phpstan/issues/new                        
```

**Comment by samsonasik** [comment]
Tried phpstan with `--debug`, got the following trace: 

```bash
PHP Fatal error:  Uncaught Error: Call to a member function getAttribute() on null in /home/runner/work/rector/rector/vendor/symplify/phpstan-rules/src/Rules/CheckTypehintCallerTypeRule.php:74
Stack trace:
#0 /home/runner/work/rector/rector/vendor/symplify/phpstan-rules/src/Rules/AbstractSymplifyRule.php(49): Symplify\PHPStanRules\Rules\CheckTypehintCallerTypeRule->process(Object(PhpParser\Node\Expr\MethodCall), Object(PHPStan\Analyser\MutatingScope))
#1 phar:///home/runner/work/rector/rector/vendor/phpstan/phpstan/phpstan/src/Analyser/FileAnalyser.php(70): Symplify\PHPStanRules\Rules\AbstractSymplifyRule->processNode(Object(PhpParser\Node\Expr\MethodCall), Object(PHPStan\Analyser\MutatingScope))
#2 phar:///home/runner/work/rector/rector/vendor/phpstan/phpstan/phpstan/src/Node/ClassStatementsGatherer.php(91): PHPStan\Analyser\FileAnalyser->PHPStan\Analyser\{closure}(Object(PhpParser\Node\Expr\MethodCall), Object(PHPStan\Analyser\MutatingScope))
#3 phar:///home/runner/work/rector/rector/vendor/phpstan/phpstan/phpstan/src/Analys in /home/runner/work/rector/rector/vendor/symplify/phpstan-rules/src/Rules/CheckTypehintCallerTypeRule.php on line 74
Fatal error: Uncaught Error: Call to a member function getAttribute() on null in /home/runner/work/rector/rector/vendor/symplify/phpstan-rules/src/Rules/CheckTypehintCallerTypeRule.php:74
Stack trace:
#0 /home/runner/work/rector/rector/vendor/symplify/phpstan-rules/src/Rules/AbstractSymplifyRule.php(49): Symplify\PHPStanRules\Rules\CheckTypehintCallerTypeRule->process(Object(PhpParser\Node\Expr\MethodCall), Object(PHPStan\Analyser\MutatingScope))
#1 phar:///home/runner/work/rector/rector/vendor/phpstan/phpstan/phpstan/src/Analyser/FileAnalyser.php(70): Symplify\PHPStanRules\Rules\AbstractSymplifyRule->processNode(Object(PhpParser\Node\Expr\MethodCall), Object(PHPStan\Analyser\MutatingScope))
#2 phar:///home/runner/work/rector/rector/vendor/phpstan/phpstan/phpstan/src/Node/ClassStatementsGatherer.php(91): PHPStan\Analyser\FileAnalyser->PHPStan\Analyser\{closure}(Object(PhpParser\Node\Expr\MethodCall), Object(PHPStan\Analyser\MutatingScope))
#3 phar:///home/runner/work/rector/rector/vendor/phpstan/phpstan/phpstan/src/Analys in /home/runner/work/rector/rector/vendor/symplify/phpstan-rules/src/Rules/CheckTypehintCallerTypeRule.php on line 74
```

It seems really on this patch https://github.com/symplify/symplify/pull/2564

**Pull request merged**